RoaringBitmap/src/main/java/org/roaringbitmap/memory/DefaultMemoryManager.java

```diff
@@ -0,0 +1,35 @@
+package org.roaringbitmap.memory;
+
+/**
+ * Default simple implementation for IMemoryManager where create methods creates the char/long array
+ * and the free method does nothing.
+ */
+public final class DefaultMemoryManager implements IMemoryManager {
```

Comment:
This prevents leaks from occurring in existing code but referencing it throughout the data structure still imposes a footprint overhead on existing users.

Reply:
Is there a way to measure the footprint? Other than the obvious 8 bytes per container/bitmap

Reply:
JOL reports the class layout, it should be measured with and without compressed references.